### DISCUSSION TITLE
Смещение по центру.

### DISCUSSION THREAD

**Discussion by nickson-82** [zhongyang219/TrafficMonitor#1109] [Q&A]
После последнего обновления 22H2 22581.100 Tasbar сместился по центру  панели задач, переместить обратно вправо не получается, как исправить данную проблему ? Пере пробывал все настройки,  не чего не выходит.
![Снимок экрана 2022-03-27 104341](https://user-images.githubusercontent.com/102458763/160272003-13477f2c-d6f5-4412-b5f8-f188eab2e27c.png)

**Comment by zbwlalala** [comment]
![Uploading QQ截图20220408152830.png…]()
I met the same problem.

**Comment by lerosed** [comment]
I have that too.
